versioned_docs/version-23.04/monitoring/basic-objects/hosts.md

```diff
@@ -6,128 +6,132 @@ title: Creating hosts manually
 To create a host manually:
 
 1. Go to **Configuration \> Hosts \> Hosts** and then click **Add**.
-2. Fill in the fields (see [below](#host-configuration-tab)), then click on **Save**.
+2. Fill in the fields (see [below](#host-configuration-tab)), and then click **Save**.
 3. [Deploy the configuration](../monitoring-servers/deploying-a-configuration.md).
 
 ## Host configuration tab
 
 ### Host basic information
 
-* The **Name** field defines the host name that will be used by the Monitoring Engine. The following characters are not authorized: `~!$%^&|'"<>?,()=*{}` and spaces.
-* The **Alias** field shows the alias of the host.
-* The **Address** field defines IP address or DNS name of the host. The **Resolve** button enables us to
-  resolve the domain name by questioning the DNS server configured on the central server.
-* The **SNMP Community & Version** fields contain the name of the community and the SNMP version.
-* The **Monitoring server** field indicates which poller server is charged with monitoring this host.
-* The **Timezone** field indicates the timezone location of the monitored hosts.
-* The **Templates** field enables us to associated one or more templates of hosts with this object.
+* **Name**: host name used by the Monitoring Engine. Cannot contain `~!$%^&|'"<>?,()=*{}` and spaces will be automatically replaced with _.
+* **Alias**: another name for the host. Spaces and characters unauthorized in the name are allowed here. You can also use the alias to look up the host using the Search bar in the Resource Status page with the syntax `alias:`.
+* **Address**: IP address or DNS name of the host. The **Resolve** button tests the domain name by questioning the DNS server configured on the central server. If given a DNS name, the **Resolve** button will also replace the text with the matching IP address.
+* **SNMP Community & Version**: name of the community that you have defined for the equipment and its version. If the version is 1 or 2c, enter the community's name in the first field. If the version is 3, leave the first field empty and fill in the [`snmpextraoptions`](/pp/integrations/plugin-packs/getting-started/how-to-guides/troubleshooting-plugins/#snmpv3-options-mapping) custom macro that will automatically appear in the **Host check options** section.
```

Comment:
unlike the French version you do not indicate that the macro comes with an SNMP template.